lib/layer/resource.rb

```diff
@@ -63,7 +63,8 @@ def find(client, url, id)
       end
 
       def list(client, url, params = {})
-        collection = client.get(url, body: params.to_json)
+        url = "#{url}?#{params.to_query}" if params.any?
```

Comment:
`to_query` won't work for projects that aren't using rails or active support.

My suggestion would be something along the lines of:
```
if params.any?
  query = params.collect{|key, val| "#{key}=#{val}" }.join("&")
  url = "#{url}?#{query}"
end
```
What do you think?